docs/General/buildRPC.md

```diff
@@ -11,7 +11,7 @@ After running the RPC, the Intel® AMT managed device can be managed remotely us
 **Figure 1: RPC Configuration** 
 
 !!! tip "Figure 1 Details"
-         The RPC on the managed devices communicates with the Manageability Engine Interface (MEI) and Remote Provisioning Server (RPS) interfaces. The MEI uses the ME Driver to talk to Intel® AMT (Steps 3 and 4, Figure 1). The RPC activates Intel® AMT with a control mode profile, which is associated with a CIRA configuration. The profile, Client Control Mode (CCM) or Admin Control Mode (ACM), and configuration were created in [Create a CIRA Config](../General/createCIRAConfig.md) or [Create an AMT Profile](../General/createProfileACM.md). After running RPC with a profile, the MPS can manage the remote device and issue AMT commands (Steps 5, Figure 1).
+    The RPC on the managed devices communicates with the Manageability Engine Interface (MEI) and Remote Provisioning Server (RPS) interfaces. The MEI uses the ME Driver to talk to Intel® AMT (Steps 3 and 4, Figure 1). The RPC activates Intel® AMT with a control mode profile, which is associated with a CIRA configuration. The profile, Client Control Mode (CCM) or Admin Control Mode (ACM), and configuration were created in [Create a CIRA Config](../General/createCIRAConfig.md) or [Create an AMT Profile](../General/createProfileACM.md). After running RPC with a profile, the MPS can manage the remote device and issue AMT commands (Steps 5, Figure 1).
```

Comment:
I actually formed this paragraph in a one my last few checkins. I was wondering if it sounds accurate.  I hope so. 

Steps 5 should be Step 5.

Reply:
I took a stab at slightly modified terminology if you can take a look.